bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/Runtime.java

```diff
@@ -145,15 +146,17 @@ public BFuture invokeMethodAsyncConcurrently(BObject object, String methodName,
      * for the mutable state with given arguments, use @invokeMethodAsyncConcurrently
      * otherwise @invokeMethodAsyncSequentially .
      * <p>
-     * We can decide the object method isolation by using object.getType().isIsolated(methodName).
+     * We can decide the object method isolation by using both object.getType().isIsolated() and
+     * object.getType().isIsolated(methodName).
      */
     @Deprecated
     public BFuture invokeMethodAsync(BObject object, String methodName, String strandName, StrandMetadata metadata,
```

Comment:
Shall we add same doc comments to invokeMethodAsyncSequentially and invokeMethodAsyncConcurrenlt method as well